addons/police/functions/fnc_copKilledMarker.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function creates marker on cop's death position and handles marker decay.
+ *
+ * Arguments:
+ * 0: Killed unit <OBJECT>
+ * 1: Unit killed time hh:mm:ss <STRING>
+ *
+ * Return Value:
+ * 0: Created marker <STRING>
+ *
+ * Example:
+ * [player, "08:09:10"] call afsk_police_fnc_copKilledMarker
+ *
+ * Public: No
+ */
+
+params ["_unit", "_time"];
+
+private _markerName = format ["killed_cop_%1", random (999)];
```

Comment:
```suggestion
private _markerName = format ["killed_cop_%1", _unit];
```

Reply:
Will this work if some cop is killed more than once?

Reply:
If he is killed before the marker expires and is deleted then not.
could always do:
```
private _markerName = format ["killed_cop_%1_%2", _unit, time];